core/themes/basis/config/basis.settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "_config_name": "basis.settings",
+    "default_version_clean": "1.30",
```

Comment:
Are we going to need to remember to change this with every core release? If so, is there a better way to manage this? Can we do it in an install hook instead?

Reply:
> Are we going to need to remember to change this with every core release?

Every core release where we add another value to `basis_supplemental_css_versions()`, yes. But we won't update it on releases that don't have CSS changes.

Reply:
>  To some extent, it wouldn't or won't be necessary at all, if I can figure out why the PHP version_compare funtion isn't working for me.

This extra variable can be eliminated. I filed a PR at https://github.com/stpaultim/backdrop/pull/8 to remove it.